strats.md

```diff
@@ -520,6 +656,32 @@ A `comeInJumping` entrance condition represents the need for Samus to be able to
 }
 ```
 
+### Come In Space Jumping
+
+A `comeInSpaceJumping` entrance condition indicates that Samus must come in with a Space Jump through the bottom of the doorway, applicable to horizontal transitions. It has the following properties:
+
+* _speedBooster_: If true, then Speed Booster must be used while gaining run speed. If false, then Speed Booster must not be used. If "any", then Speed Booster may or may not be used.
+* _minTiles_: The minimum horizontal speed that will satisfy the condition, measured in effective runway tiles with dash held on the remote runway.
+* _maxTiles_: The maximum horizontal speed that will satisfy the condition, measured in effective runway tiles with dash held on the remote runway.
+
+A `comeInSpaceJumping` entrance condition must match with a `leaveSpaceJumping` on the other side of the door. To match, the `blue` property of `leaveSpaceJumping` must be "no" or "any". This comes with an implicit `SpaceJump` item requirement.
```

Comment:
Correct, for now at least. I didn't foresee an immediate need for combining the late bottom-of-door Space Jump with blue speed, so there's no entrance condition for that. But I figured it's easy enough to include as a possibility in the exit condition, in case it comes up later.